packages/home-spa/src/js/controller.js

```diff
@@ -26,9 +26,10 @@ if (microserviceCard !== null && carouselSlide !== null) {
 
 getData().then(
   (data) => {
-    buildDom(data);
+    buildDom(data.data.listHomeType);
```

Comment:
Change the argument name to something else, like `result` or something, as the data is being repeated.

Reply:
Change applied